app/helpers/application_helper.rb

```diff
@@ -61,4 +61,14 @@ def software_color_class(software_version)
       'text-danger'
     end
   end
+
+  def transaction_type_icon(transaction_type_string)
+    case transaction_type_string
+    when "transfer"
+      '<i class="fas fa-exchange-alt text-success mr-1"></i>'
+    # TODO see: POSSIBLE_TRANSACTION_TYPES
```

Comment:
This comment is not releavant anymore. Can be removed. But separate icon for this type (and some other types) is okay I think.